src/migration/1614674910823-AddSeasonColumnsToProduces.ts

```diff
@@ -0,0 +1,15 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class AddSeasonColumnsToProduces1614674910823 implements MigrationInterface {
+  name = "AddSeasonColumnsToProduces1614674910823";
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE "Produces" ADD "seasonFirstMonth" smallint NOT NULL DEFAULT '0'`);
+    await queryRunner.query(`ALTER TABLE "Produces" ADD "seasonMonthsCount" smallint NOT NULL DEFAULT '0'`);
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE "Produces" DROP COLUMN "seasonMonthsCount"`);
+    await queryRunner.query(`ALTER TABLE "Produces" DROP COLUMN "seasonFirstMonth"`);
+  }
```

Comment:
@camfrelat juste pour ta "culture". Ceci est du code faisant une migration du schéma de la base de donnée.

Elle est automatiquement générée par les changement que j'ai fait dans la table `produces` (l'autre fichier modifié).

Le text dedans (`ALTER TABLE "Produces" ADD "seasonFirstMonth" smallint NOT NULL DEFAULT '0'`) est du language SQL :)

Reply:
okay